modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavascriptEs6AngularGenerator.java

```diff
@@ -0,0 +1,191 @@
+package io.swagger.codegen.languages;
```

Comment:
Add ES6 as an option instead of introducing a new generator
Remove `/src/main/java/io/swagger/codegen/languages/JavascriptEs6AngularGenerator.java`
and append `es6` as a cli option to `/src/main/java/io/swagger/codegen/languages/JavascriptAngularGenerator.java` 

But, it would make sense to keep the templates separated.